src/components/PeopleCard.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import clsx from "clsx";
+import { Link } from "react-router-dom";
+import styles from "./people.module.css";
+const PeopleCard = (props) => {
+  console.log(props);
+  return (
+    <div className={clsx("card-demo", styles.card)}>
+      <div className="card">
+        <div className="card__header">
+          <div className="avatar">
+            <div className="avatar__intro">
+              <div className="avatar__name">{props.name}</div>
+              {/* <small className="avatar__subtitle">{props.role}</small> */}
```

Comment:
I think this line is redundant

Reply:
Actually role is another property of the contributor object.

Reply:
Okay